packages/admin/admin/src/common/Accordion.tsx

```diff
@@ -0,0 +1,98 @@
+    accordionSummary?: Partial<AccordionSummaryProps>;
+    accordionDetails?: Partial<AccordionDetailsProps>;
+}
+interface AccordionProps {
+    title: React.ReactNode;
+    supportText?: React.ReactNode;
+    endAdornment?: React.ReactNode;
+    initialExpanded?: boolean;
+    componentsProps?: AccordionComponentsProps;
+}
+
+export const Accordion: React.FC<AccordionProps> = ({ title, supportText, endAdornment, children, initialExpanded = false, componentsProps }) => {
+    const [expanded, setExpanded] = React.useState(initialExpanded);
+
+    const handleChange = (event: React.SyntheticEvent, isExpanded: boolean) => {
+        setExpanded(isExpanded);
+    };
+
+    return (
+        <AccordionComponent expanded={expanded} onChange={handleChange} disableGutters elevation={0} square>
+            <AccordionSummary {...componentsProps?.accordionSummary}>
+                <AccordionColumn>
+                    <AccordionTitleText expanded={expanded}>{title}</AccordionTitleText>
+                    <AccordionSupportText>{supportText}</AccordionSupportText>
+                </AccordionColumn>
+                <FillSpace />
+                <AccordionEndAdornment>{endAdornment}</AccordionEndAdornment>
+            </AccordionSummary>
+            <AccordionDetails {...componentsProps?.accordionDetails}>{children}</AccordionDetails>
+        </AccordionComponent>
+    );
+};
+
+const AccordionComponent = styled(MuiAccordion)(({ theme }) => ({
+    border: `1px solid ${theme.palette.divider}`,
+    "&:not(:last-child)": {
+        marginBottom: "20px",
+    },
+    "&:before": {
+        display: "none",
+    },
+}));
+
+const AccordionSummary = styled((props: AccordionSummaryProps) => <MuiAccordionSummary expandIcon={<ArrowForwardIosSharpIcon />} {...props} />)`
+    display: flex;
+    flex-direction: row-reverse;
+    padding: 0 20px;
+    gap: 20px;
+    height: 86px;
+`;
+
+const AccordionEndAdornment = styled(MuiAccordionSummary)`
+    align-items: center;
+`;
+
+const AccordionDetails = styled(MuiAccordionDetails)(({ theme }) => ({
+    display: "flex",
+    flexDirection: "column",
+    padding: "40px",
+    borderTop: `1px solid ${theme.palette.divider}`,
+}));
+
+const AccordionColumn = styled(MuiAccordionDetails)`
+    flex-direction: column;
+    padding: 10px;
+`;
+
+const FillSpace = styled("div")`
+    flex-grow: 1;
+    box-sizing: inherit;
+    user-select: none;
+`;
+
+interface AccordionTitleTextProps {
+    expanded: boolean;
+}
+
+const AccordionTitleText = styled("div")<AccordionTitleTextProps>`
+    font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+    font-size: 16px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: ${({ expanded, theme }) => (expanded ? theme.palette.primary.main : theme.palette.text.primary)};
+`;
+
+const AccordionSupportText = styled("div")`
+    font-size: 12px;
+    line-height: 18px;
+    color: ${({ theme }) => theme.palette.text.secondary};
+`;
```

Comment:
Does adding some of these styles to their respective style-overrides in the theme make sense? 
Like we have done for `MuiPaper` for instance: https://github.com/vivid-planet/comet/blob/main/packages/admin/admin-theme/src/componentsTheme/MuiPaper.ts#L10

Then, for example, `MuiAccordion` would always have a 1px border, also when used without the `Accordion` from `@comet/admin`.